### PR TITLE
Documentation centos 7 and debian 8

### PR DESCRIPTION
**Pull request by fdurand**
# Description

Updated documentation for centos 7 and debian 8
# Delete branch after merge

YES

### CODE REVIEW

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 1]
```diff
@@ -326,6 +325,431 @@ Create the pem that combines the key and certificate for the http services. Adap
...
+
+Next, you need to remove the empty users from your MySQL database
+
+----
+# mysql
+mysql> delete from mysql.user where user = '' ;
+mysql> flush privileges;
+----
+
+Installation on Centos 7
```
Comment (extrafu) [inline docs/PacketFence_Clustering_Guide.asciidoc]: CentOS

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 2]
```diff
@@ -326,6 +325,431 @@ Create the pem that combines the key and certificate for the http services. Adap
...
+------------------------
+
+First make sure that you system is correctly configured and uptodate
```
Comment (extrafu) [inline docs/PacketFence_Clustering_Guide.asciidoc]: up-to-date

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 3]
```diff
@@ -326,6 +325,431 @@ Create the pem that combines the key and certificate for the http services. Adap
...
+
+  # yum update
+
+`/etc/hosts` must contain:
+
+  ha_interface_ip_address_1 pf1_server_name
+  ha_interface_ip_address_2 pf2_server_name
+
+Set hostname on each servers:
```
Comment (extrafu) [inline docs/PacketFence_Clustering_Guide.asciidoc]: on each server:

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 4]
```diff
@@ -326,6 +325,431 @@ Create the pem that combines the key and certificate for the http services. Adap
...
+
+[options="compact"]
+* `pf1_server_name` and `pf2_server_name` by the real server names (http://en.wikipedia.org/wiki/Fully_qualified_domain_name[FQDN])
+* `ha_interface_ip_address_1` and `ha_interface_ip_address_2` by the IP addresses of the management interface on both servers.
+
+
+Step 1: Install the replicated database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+NOTE: In this implementation, the database is replicated in active/passive across two servers that are part of the PacketFence cluster. Active/active replication is also possible using MariaDB Galera cluster but this subject is not covered in this guide.
```
Comment (extrafu) [inline docs/PacketFence_Clustering_Guide.asciidoc]: In this example, ...

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 5]
```diff
@@ -326,6 +325,431 @@ Create the pem that combines the key and certificate for the http services. Adap
...
+
+Creation of the DRBD partition
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+During the OS installation, reduce the size of the main partition and create a new one (that will be used for the replicated MySQL database) of 30G. *Do not* create that partition during the install process, we will do it later.
+
+Partitioning
+^^^^^^^^^^^^
+
+After the install, you need to create the extra partition for drbd. Using fdisk, create your new partition and save the table. You will probably need to reboot your server after this step.
```
Comment (extrafu) [inline docs/PacketFence_Clustering_Guide.asciidoc]: After the installation, ... for DRBD.

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 6]
```diff
@@ -326,6 +325,431 @@ Create the pem that combines the key and certificate for the http services. Adap
...
+ gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-elrepo.org
+ protect=0
+----
+
+Install now the package DRBD v8.4 enabling .
+
+  # yum install kmod-drbd84 --enablerepo=elrepo
+
+
+CAUTION: Initializing, configuring and troubleshooting DRBD is not straight forward! We strongly recommend that you read the online documentation available on DRBD website so you have a better idea about how it works.
```
Comment (extrafu) [inline docs/PacketFence_Clustering_Guide.asciidoc]: on DRBD website -> from the DRBD website

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 7]
```diff
@@ -326,6 +325,431 @@ Create the pem that combines the key and certificate for the http services. Adap
...
+
+Here we assume the name of the partition is mysql.
```
Comment (extrafu) [inline docs/PacketFence_Clustering_Guide.asciidoc:477]: `mysql`

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 8]
```diff
@@ -326,6 +325,431 @@ Create the pem that combines the key and certificate for the http services. Adap
...
+
+Load the DRBD kernel module:
+
+  modprobe drbd
+
+Edit `/etc/drbd.d/global_common.conf` and put the following content:
```
Comment (extrafu) [inline docs/PacketFence_Clustering_Guide.asciidoc]: and put -> with

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 9]
```diff
@@ -326,6 +325,431 @@ Create the pem that combines the key and certificate for the http services. Adap
...
+
+  # systemctl stop mariadb
+  # mkdir /var/lib/mysql.bak
+  # mv /var/lib/mysql/* /var/lib/mysql.bak/
+
+Mount the partition with:
+
+  # mount /dev/drbd0 /var/lib/mysql
+
+Start MariaDBL
```
Comment (extrafu) [inline docs/PacketFence_Clustering_Guide.asciidoc]: DBL -> DB

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 10]
```diff
@@ -326,6 +325,431 @@ Create the pem that combines the key and certificate for the http services. Adap
...
+Step 2 : Server configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You will need to configure the server so the services can bind on IP addresses they don't currently have configured. This allows faster failover of the services.
+
+On CentOS, add the following line in /etc/sysctl.conf and then reload with 'sysctl -p'
+
+  net.ipv4.ip_nonlocal_bind = 1
+
+Create the pem that combines the key and certificate for the http services. Adapt to your own paths if you are using different certificates.
```
Comment (extrafu) [inline docs/PacketFence_Clustering_Guide.asciidoc]: Create the PEM file

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 11]
```diff
@@ -262,7 +247,7 @@ RHEL / CentOS
 
 In order to use the PacketFence repository :
 
-  # rpm -Uvh http://packetfence.org/downloads/PacketFence/RHEL6/`uname -i`/RPMS/packetfence-release-1-2.centos6.noarch.rpm
+  # rpm -Uvh http://packetfence.org/downloads/PacketFence/RHEL6/`uname -i`/RPMS/packetfence-release-1.2-5.1.noarch.rpm
```
Comment (julsemaan) [inline docs/PacketFence_Administration_Guide.asciidoc]: will that file give the right config for RHEL7 as well ?

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 12]
```diff
@@ -318,14 +317,48 @@ On CentOS, add the following line in /etc/sysctl.conf and then reload with 'sysc
...
-Create the pem that combines the key and certificate for the http services. Adapt to your own paths if you are using different certificates.
+Create the PEM that combines the key and certificate for the http services. Adapt to your own paths if you are using different certificates.
 
   # cd /usr/local/pf/conf/ssl
   # cat server.key server.crt > server.pem
 
 MySQL configuration
 ^^^^^^^^^^^^^^^^^^^
 
+ node pf1_server_name \
```
Comment (julsemaan) [inline docs/PacketFence_Clustering_Guide.asciidoc]: Not sure that goes there.

This is a corosync configuration, not a MySQL one

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 13]
```diff
@@ -361,6 +394,399 @@ mysql> delete from mysql.user where user = '' ;
...
+
+  # hostnamectl set-hostname pf1_server_name
+  # hostnamectl set-hostname pf2_server_name
+
+Disable selinux
+
+  # setenforce 0
+
+[options="compact"]
+* `pf1_server_name` and `pf2_server_name` by the real server names (http://en.wikipedia.org/wiki/Fully_qualified_domain_name[FQDN])
```
Comment (julsemaan) [inline docs/PacketFence_Clustering_Guide.asciidoc:383]: Do you think @mamadou on the ML would get it, even with that much details ?

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 14]
```diff
@@ -361,6 +394,399 @@ mysql> delete from mysql.user where user = '' ;
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+During the OS installation, reduce the size of the main partition and create a new one (that will be used for the replicated MySQL database) of 30G. *Do not* create that partition during the install process, we will do it later.
+
+Partitioning
+^^^^^^^^^^^^
+
+After installation, you need to create the extra partition for DRBD. Using fdisk, create your new partition and save the table. You will probably need to reboot your server after this step.
+
+
```
Comment (julsemaan) [inline docs/PacketFence_Clustering_Guide.asciidoc]: Remove the extra lines :nail_care:

File: docs/PacketFence_Administration_Guide.asciidoc (hunk 11 above)
Comment (fdurand) [inline docs/PacketFence_Administration_Guide.asciidoc]: yes

File: docs/PacketFence_Clustering_Guide.asciidoc (hunk 12 above)
Comment (fdurand) [inline docs/PacketFence_Clustering_Guide.asciidoc]: Arf, copy and paste issue

**Pull request merged**